CONTRIBUTING.md

```diff
@@ -19,7 +19,8 @@ You can run the project by following these steps:
 1. Clone this repository.
 2. `docker-compose up -d` - Run Redis and MariaDB
 3. `npm install`
-4. Run `npm run start:dev`
+4. `npm run build`
```

Comment:
If this is because of the data files, let's change 3. to `npm install && npm run download:files` to make it clear it's a one- time setup. The rest of build shouldn't be needed.